lib/aarch64/parser.y

```diff
@@ -1057,6 +1068,21 @@ rule
       }
     ;
 
+  reg_imm_imm_or_label
```

Comment:
I'm intending that the user reads the last two values as being grouped, i.e.`reg_imm_imm_or_label ` should really parsed as `reg_imm_(imm_or_label)`. Happy to follow an alternative naming convention

Reply:
I think this is fine for now.